libs/features/google-login.js

```diff
@@ -3,10 +3,11 @@ const GOOGLE_ID = '530526366930-l874a90ipfkn26naa71r010u8epp39jt.apps.googleuser
 const PLACEHOLDER = 'feds-googleLogin';
 const WRAPPER = 'feds-profile';
 
-const onToken = async (getMetadata, data) => {
+const onToken = async (getMetadata, data, getConfig) => {
   let destination;
+  const config = getConfig();
   try {
-    destination = new URL(getMetadata('google-login-redirect'))?.href;
+    destination = typeof config.googleLoginURLCallback === 'function' ? new URL(await config.googleLoginURLCallback()) : new URL(getMetadata('google-login-redirect'))?.href;
```

Comment:
might be a nit: you should return the `href` string and based on that info you can rewrite this line as:
```suggestion
    destination = new URL(typeof config.googleLoginURLCallback === 'function' ? await config.googleLoginURLCallback() : getMetadata('google-login-redirect'))?.href;
```

Reply:
oh thanks, I think it was more than just a nit tbh. Done!